keps/sig-node/3008-qos-class-resources/README.md

```diff
@@ -512,11 +576,36 @@ Consider including folks who also work outside the SIG or subproject.
 
 The detailed design presented here covers [implementation phase 1](#phase-1)
 (see [implementation phases](#implementation-phases) for an outline of all
-planned changes).
+planned changes). Configuration and management of the QoS-class resources is
+fully handled by the underlying container runtime and is invisible to kubelet.
+
+Summary of the proposed design details:
+
+- QoS resources are opaque (just names) to kubernetes, configuration and
```

Comment:
nit: I'd say "handled in the nodes (kubelet, container runtime, etc)" which leaves room for the idea of kubelet-defined resources and classes

Reply:
Changed to *...handled in the node (container runtime, kubelet)...*

Reply:
solved (I think) @thockin

Reply:
ping @thockin, do you think this is resolved?